src/windows/animated_title.py

```diff
@@ -132,4 +132,4 @@ def clear_effect_controls(self):
                 self.settingsContainer.layout().removeWidget(child)
                 child.deleteLater()
             except:
-                pass
+                log.warning('Failed to remove child widget for effect controls')
```

Comment:
Codacy (and any Python linter) is even more pissy about bare `except:` than it is about `try/except/pass'! This will never fly unless it's at least:
```python
except Exception:
    log.something(something something dark side something exception)
```
But I would argue that the exception is, again, nothing the user can do anything about, so "warning" doesn't make sense. My preference would be:
```python3
except Exception as ex:
    log.debug("Failure removing Blender parameter widget: {}".format(ex))
```